fineract-command/src/main/resources/db/custom-changelog/0001_command_init.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="fineract" id="command-00001" context="postgresql">
+        <createTable tableName="m_command">
+            <column autoIncrement="true" name="id" type="bigint" remarks="Internal ID">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="created_at" type="datetime" />
```

Comment:
Use `TIMESTAMP WITH TIME ZONE` instead?

Reply:
I followed other examples in our migration scripts. Sure, can change... but what's the reason for the others to be `datetime` then?

Reply:
![image](https://github.com/user-attachments/assets/0f3e896a-f6b9-4b0d-a50b-659b951b940f)

Quick search in IntelliJ reveals quite some places where exactly the same is used.